### ISSUE TITLE
Unable to run InaSAFE

### ISSUE THREAD

**Issue by talitharahmawati** [inasafe/inasafe#4761]
### The run button is not working

Hi, I found a problem to run the application. I have filled all the necessary keyword wizards but seems like the run option is still not working.
Copy/paste some InaSAFE logs here if it was an error.
![image](https://user-images.githubusercontent.com/29425952/33796377-ac35b540-dd25-11e7-994c-09d1e01b35df.png)

### Environment
* InaSAFE : 4.2.5
* QGIS : 2.18
* OS : Windows
* Data used (filenames and paths so as to reproduce, a project is better):
  * Exposure : 
  * Hazard : 
  * Aggregation :
  * Extent : 
  
Checklist (if applicable): 
* [ ] Did you try to run it in debug mode ?
* [ ] Is the data available ?
  * If yes, where (BTSync, data.inasafe.org, zip) ?

**Comment by Gustry** [comment]
Can pan/zoom the map?
It will trigger an event in InaSAFE and you should have a message in the dock?

Your last action might be a click on a layer to display its keywords, so it's hidding the status of the impact function. For instance, the message might be incompatible exposure/hazard, or not overlapping layers.

**Comment by talitharahmawati** [comment]
@Gustry Yes, I can zoom the map. 
I think the problem is because one layer is not compatible to other layers, since I downloaded data from other sources ( ArcGIS online) and apparently it has different projection system, so makes it on the fly.
My other question is, how can we make this layer compatible? I have changed it to current projection system (CRS EPSG 4326) but still not working.

Note : I downloaded administration layer from ArcGIS online as the analysis extent

**Comment by ismailsunni** [comment]
Hi @talitharahmawati 
Can you share the data (is possible/allowed) ? So that we can try here.

I think we have already solve the projection problem (all projection should work, CMIIW @Gustry)

**Comment by Gustry** [comment]
Sorry I wasn't clear maybe. If you pan/zoom, the inasafe dock is not updated with a message?

Yes it's not a problem with projections. We support On the Fly Projections

**Comment by talitharahmawati** [comment]
@ismailsunni @Gustry 
Here I screen shoot the difference if I use administration boundaries and not : 
![image](https://user-images.githubusercontent.com/29425952/33892149-a4f0fd28-df8a-11e7-97fe-002187d35b9e.png)
![image](https://user-images.githubusercontent.com/29425952/33892183-bf09d69e-df8a-11e7-9230-54581dd8ef04.png)
As you can see, it only can run if I don't select the administration boundary

**Comment by Gustry** [comment]
Yes, because your aggregation is invalid, according to the error message displayed. So InaSAFE won't be able to use it, so the 'Run' button is disabled.

You can try to use the geometry checker in QGIS (in the processing toolbox) to check what kind of errors you have in your layer.
